camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -88,7 +90,7 @@ public static void beforeClass() throws IOException {
 
     // You can't delete messages in Kafka so just writing a set of known messages that can be used for testing
     messagesWritten = new HashMap<String, List<MyMessage>>();
-    messagesWritten.put(TOPIC_1, writeKafka(TOPIC_1, 2));
+    messagesWritten.put(TOPIC_1, writeKafka(TOPIC_1, 10));
```

Comment:
Change to 10 because I want to test 30% of the message skipped (3 skipped, 7 succeeded)